### PR TITLE
Add stakeeasy and update polar description

### PR DESCRIPTION
**Pull request by coco-sha**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,6 +64,7 @@ Large products that launched (or will launch very soon) on the Secret Network
     * **Silk** - Algorithmic stablecoin with transactional privacy by default
     * **Synthetics** - Algorithmic synthetic assets with transactional privacy by default
     * Many more will be revealed :shushing_face:
+* :soon: [StakeEasy](https://stakeeasy.finance/) - Privacy preserving liquid staking solution for secret network.
```
Comment (reuvenpo) [inline README.md]:
```suggestion
* :soon: [StakeEasy](https://stakeeasy.finance/) - Privacy preserving liquid staking solution for Secret Network.
```

File: README.md [hunk 2]
```diff
@@ -198,7 +199,7 @@ Tools for interacting with the secret network and for writing Secret Contracts
 * [SecretJS](https://www.npmjs.com/package/secretjs) - JavaScript library for interacting with the network
     ([examples](https://github.com/enigmampc/SecretJS-Templates))
 * [Griptape.js](https://griptapejs.com/) - By Validator [Stake or Die][stakeordie], a front-end web framework for developing DApps on the network
-* [Polar](https://www.npmjs.com/package/secret-polar) - CLI for easier development of secret contracts
+* [Polar](https://www.npmjs.com/package/secret-polar) - Polar is a development framework for building secret contracts. It facilitates features such as initiating project repo from [contract templates](https://github.com/arufa-research/polar-templates), easy compilation of contracts, deployment, Interacting with contracts using schema and contract testing framework.
```
Comment (reuvenpo) [inline README.md]: This line should ideally be an "executive summary" of the project, and ideally shouldn't be longer than one line when rendered in the main page. That being said, the additional information is useful. Besides that, you have a couple minor typos here.

So, I think we can either just fix the typos:
```suggestion
* [Polar](https://www.npmjs.com/package/secret-polar) - Polar is a development framework for building secret contracts. It facilitates features such as initializing project repos from [contract templates](https://github.com/arufa-research/polar-templates), easy compilation of contracts, deployment, Interacting with contracts using their schema, and a contract testing framework.
```

Or at least condense the information to make it somewhat shorter:
```suggestion
* [Polar](https://www.npmjs.com/package/secret-polar) - Polar is a framework for building secret contracts. It features facilities for [project templates](https://github.com/arufa-research/polar-templates), compilation, deployment, javascript interaction (based on schemas), and testing
```

**Review by reuvenpo — CHANGES_REQUESTED** [review]
Hi Coco! Thanks for this PR! I added a couple comments about typos or minor changes

File: README.md (hunk 2 above)
Comment (coco-sha) [reply README.md]: got it. will condense the information

File: README.md [hunk 3]
```diff
@@ -198,7 +199,7 @@ Tools for interacting with the secret network and for writing Secret Contracts
 * [SecretJS](https://www.npmjs.com/package/secretjs) - JavaScript library for interacting with the network
     ([examples](https://github.com/enigmampc/SecretJS-Templates))
 * [Griptape.js](https://griptapejs.com/) - By Validator [Stake or Die][stakeordie], a front-end web framework for developing DApps on the network
-* [Polar](https://www.npmjs.com/package/secret-polar) - CLI for easier development of secret contracts
+* [Polar](https://www.npmjs.com/package/secret-polar) - Polar is a framework for building secret contracts. It features facilities for [project templates](https://github.com/arufa-research/polar-templates), compilation, deployment, javascript interaction (based on schemas), and testing
```
Comment (coco-sha) [inline README.md]:
```suggestion
* [Polar](https://www.npmjs.com/package/secret-polar) - Polar is a framework for building secret contracts. It features facilitates for [project templates](https://github.com/arufa-research/polar-templates), compilation, deployment, javascript interaction (based on schemas), and testing
```

**Comment by coco-sha** [comment]
Hi Reuven! Thanks for the review! I have updated the PR.

**Review by reuvenpo — APPROVED** [review]
Looks great, thanks!

**Comment by reuvenpo** [comment]
I'll merge this now but notice that some of your commits are not signed. You should make sure that your local git client signs your commits so that no one can impersonate your identity in various repositories

**Pull request merged**